packages/aws-cdk-lib/core/lib/fs/options.ts

```diff
@@ -73,6 +73,14 @@ interface FileOptions {
    * @default IgnoreMode.GLOB
    */
   readonly ignoreMode?: IgnoreMode;
+
+  /**
+   * File paths matching the patterns will be included, based on simple glob patterns.
+   * Has no effect on Assets bundled using the `bundling` property.
+   *
+   * @default - everything is included
+   */
+  readonly include?: string[];
```

Comment:
I think it would help to add some more detail here.
1. How does this interact with `exclude`? That is, if a file matches something in `include` and `exclude`, what happens? My assumption is that `exclude` takes priority but it would be nice to mention that explicitly.
2. How does this interact with symlinks when using a `SymlinkFollowMode` other than `NEVER`? That is, if I include the symlink but not its target, what happens?

It would also be good to add test cases for these points.

Reply:
@rittneje 

Thanks for review! I added the docs and tests.

1. `exclude` takes priority if both `include` and `exclude` are matched.
    - https://github.com/aws/aws-cdk/pull/26365/commits/5dedfcb1c8825051705353276c555b6ba02a6e56 
2. If matching the symlink but not its target, it is included even if `SymlinkFollowMode` is `NEVER` or not. This is based on the same checking the `exclude` patterns.
    - https://github.com/aws/aws-cdk/pull/26365/commits/bf41fa98defe0031267f4b585019a9483c71ff59